### ISSUE TITLE
chat.js - January 2019 🎉

### ISSUE THREAD

**Issue by ongmin** [SingaporeJS/talk.js#18]
### 📝 Details
- [ ] Event: [chat.js]()
- [ ] Date: 23rd January (Wed)
- [ ] Hosts: @timoxley @ongmin
- [ ] Venue: SP Digital

### 📅 Agenda

Time   | Agenda
------ | :-----
7.00pm | Doors open - _mingle with guests and speakers over 🍕_
7.30pm | Start of event - _opening segment by host_
7.40pm | Start of scheduled talks
8.40pm | Open announcements - _open segment for anyone to make announcements_
9.00pm | End of event

### :speech_balloon: Talks

- [ ] My Awesome Talk by _You_

### :seedling: Propose a talk this month

Want to speak at this month's talk? Leave a comment below with the following details:

<details>
<summary>Click for details</summary>

> #### Title of your topic
> Insert catchy title here
>
> #### Topic description
> Tell us what you'd like to talk about
>
> #### Presenter(s)
> Include GitHub handles where possible. Add your Twitter handle if you wish
>
> #### Presenter's bio
> A short introduction of yourself
>
> #### Duration
> 5 minutes for presenting an introduction to the topic, then up to 20 minutes for discussion.
>
> #### Deck URL (if any)
> You can add this part in later

</details>

### :loudspeaker: Open announcements
Have something you'd like to tell the audience? We have an open announcement segment for you to announce job opportunities, promote a meet up, upcoming conferences, etc.

---

### :pencil: Organizers' checklist

<details>
<summary>Click for details</summary>

- [x] Check available dates on [EngineersSG](https://engineers.sg/)
- [x] Confirm date and venue availability
- [x] Create event on [SingaporeJS Meetup Page](https://www.meetup.com/singapore-js/)
- [ ] Inform @engineersftw for video recordings
- [ ] Announce event on [SingaporeJS Gitter](https://gitter.im/SingaporeJS/home) and other social media platforms
- [ ] Prepare deck for event

\* _checked boxes indicate confirmed/done_

</details>

**Comment by ongmin** [comment]
Tl;dr: Chat.JS is an experimental format for a more cosy talk format with a smaller group with more exchange between the speakers and audience members. It goes like this:

1. Start off with a 5 minute lightning talk 
2. At end of talk, presenter can pose 1-2 questions to the audience to elicit opinions
3. We leave more time for this discussion and question and answer

Sounds fun? Propose a topic below. :)

**Comment by yingkh** [comment]
### Title of your topic
What's Functional Programming in Javascript?

### Topic description
The lightning talk sets the context of Functional Programming in Javascript with the discussion to understand the general acceptance and interest on Functional Programming in Javascript in the local context with the potential of having workshop in the subject matter.

### Presenter
Ying Ka Ho (@yingkh)
Twitter: @yingkh_tweets
